stdlib/public/SDK/XCTest/XCTest.swift

```diff
@@ -1030,6 +1030,28 @@ public func XCTAssertThrowsError<T>(_ expression: @autoclosure () throws -> T, _
   }
 }
 
+public func XCTAssertNoThrow<T>(_ expression: @autoclosure () throws -> T, _ message: @autoclosure () -> String = "", file: StaticString = #file, line: UInt = #line) {
+  let assertionType = _XCTAssertionType.assertion_NoThrow
+
+  let result = _XCTRunThrowableBlock { _ = try expression() }
+
+  switch result {
+  case .success:
```

Comment:
It's not enough to outdent.  If you use Xcode you can set your indentation settings appropriately, highlight these code blocks, and hit `^+I` (CNTRL+I) to have it do this for you.